core/src/actions_normalize.cpp

```diff
@@ -201,7 +200,7 @@ bool km::core::actions_normalize(
     return false;
   }
 
-  if((status = km_core_context_set(app_context, app_context_items)) != KM_CORE_STATUS_OK) {
+  if((status = km_core_context_set(static_cast<km_core_context*>(app_context), app_context_items)) != KM_CORE_STATUS_OK) {
```

Comment:
Isn't app_context already of type km_core_context*?

Reply:
It is `km::core::context` which is a class that subclasses the `km_core_context` struct. It may make more sense to bypass the public api call here, in some ways, and set the context directly, but I was trying to avoid more complex code.